src/test/scala/io/iohk/ethereum/vm/OpCodeGasSpec.scala

```diff
@@ -1,21 +1,26 @@
 package io.iohk.ethereum.vm
 
+import io.iohk.ethereum.vm.FeeSchedule.GasCost._
 import org.scalatest.{FunSuite, Matchers}
 import org.scalatest.prop.PropertyChecks
 import Generators._
 import GasFee._
 import UInt256.{One, Two, Zero}
 import io.iohk.ethereum.domain.{Account, Address}
 import io.iohk.ethereum.vm.MockWorldState.PS
-
+import scala.language.implicitConversions
 
 // scalastyle:off magic.number
 class OpCodeGasSpec extends FunSuite with OpCodeTesting with Matchers with PropertyChecks {
 
-  val stackOpsFees = (pushOps ++ dupOps ++ swapOps).map(_ -> G_verylow)
-  val constOpsFees = constOps.map(_ -> G_base)
+  override val config = EvmConfig.FrontierConfig
```

Comment:
Ideally... :)
We should first of all support the latest version, I guess 99% of our users will only need that. 
And by the way, apart from running full sync from genesis, do we have a plan for testing these changeovers?